pylint/extensions/_check_docs_utils.py

```diff
@@ -156,6 +158,109 @@ def possible_exc_types(node: nodes.NodeNG) -> Set[nodes.ClassDef]:
         return set()
 
 
+def _is_ellipsis(node: astroid.NodeNG):
+    if sys.version_info < (3, 8):
+        return isinstance(node, astroid.Ellipsis)
+
+    return isinstance(node, astroid.Const) and node.value == Ellipsis
+
+
+def _merge_annotations(annotations, comment_annotations):
```

Comment:
```suggestion
def _merge_annotations(annotations: List[nodes.?], comment_annotations: List[nodes.?]) -> Generator:
```
Not sure if we can be more specific than NodeNG here..

Reply:
I don't think that we can. The astroid annotations of the attributes that we get values from to pass into this function list them only as NodeNG at the moment.